Update to PDF.js v2.16.105

mcatanzaro opened the pull request:
#### a0d836f83a25cf6e64102c297d29a6cc0f1c4f88
<pre>
Update to PDF.js v2.16.105
<a href="https://bugs.webkit.org/show_bug.cgi?id=244497">https://bugs.webkit.org/show_bug.cgi?id=244497</a>

Reviewed by Tim Nguyen.

This also fixes an incremental build problem in PdfJSGResources.cmake.
We need to remove the dependency files when generating the resource
manifests to force them to be refreshed; otherwise, CMake uses the stale
dependency file and complains if any file disappeared when updating
PDF.js.

* Source/ThirdParty/pdfjs/PdfJSFiles.cmake:
* Source/ThirdParty/pdfjs/build/pdf.js:
(return.):
(return):
* Source/ThirdParty/pdfjs/build/pdf.sandbox.js:
* Source/ThirdParty/pdfjs/build/pdf.worker.js:
(return.):
* Source/ThirdParty/pdfjs/web/debugger.css:
(#viewer.textLayer-visible .textLayer span[aria-owns]):
* Source/ThirdParty/pdfjs/web/images/toolbarButton-editorNone.svg: Removed.
* Source/ThirdParty/pdfjs/web/locale/be/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/cs/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/cy/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/de/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/dsb/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/el/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/en-GB/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/en-US/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/eo/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/es-AR/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/es-CL/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/es-MX/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/fi/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/fr/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/fy-NL/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/gn/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/he/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/hsb/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/hu/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/ia/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/id/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/is/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/it/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/ja/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/ka/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/kab/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/kk/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/ko/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/nb-NO/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/nn-NO/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/oc/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/pa-IN/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/pl/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/pt-BR/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/pt-PT/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/ru/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/si/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/sk/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/sl/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/sv-SE/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/tg/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/th/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/tr/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/uk/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/vi/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/zh-CN/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/zh-TW/viewer.properties:
* Source/ThirdParty/pdfjs/web/viewer.css:
(.annotationLayer .popup):
(.annotationLayer .annotationTextContent):
(.annotationLayer .annotationTextContent span):
(:root):
(#editorNone::before): Deleted.
* Source/ThirdParty/pdfjs/web/viewer.html:
* Source/ThirdParty/pdfjs/web/viewer.js:
(exports.OutputScale):
(exports.OutputScale.prototype.get scaled):
(exports.scrollIntoView):
(exports.const.debounceScroll):
(exports.watchScroll):
(exports.parseQueryString):
(exports.removeNullCharacters):
(exports.binarySearchFirstItem):
(exports.approximateFraction):
(exports.roundToDivide):
(exports.getPageSizeInches):
(exports):
(exports.):
(exports.checkCanvasSizeLimitation): Deleted.
(exports.checkResizeObserver): Deleted.
(exports.AppOptions): Deleted.
(exports.AppOptions.get const): Deleted.
(exports.AppOptions.getAll): Deleted.
(exports.AppOptions.set userOptions): Deleted.
(exports.AppOptions.remove): Deleted.
(exports.AppOptions._hasUserOptions): Deleted.
(__webpack_require__.DefaultExternalServices): Deleted.
(__webpack_require__.DefaultExternalServices.updateFindControlState): Deleted.
(__webpack_require__.DefaultExternalServices.updateFindMatchesCount): Deleted.
(__webpack_require__.DefaultExternalServices.initPassiveLoading): Deleted.
(__webpack_require__.DefaultExternalServices.reportTelemetry): Deleted.
(__webpack_require__.DefaultExternalServices.createDownloadManager): Deleted.
(__webpack_require__.DefaultExternalServices.createPreferences): Deleted.
(__webpack_require__.DefaultExternalServices.createL10n): Deleted.
(__webpack_require__.DefaultExternalServices.createScripting): Deleted.
(__webpack_require__.DefaultExternalServices.get supportsIntegratedFind): Deleted.
(__webpack_require__.DefaultExternalServices.get supportsDocumentFonts): Deleted.
(__webpack_require__.DefaultExternalServices.get supportedMouseWheelZoomModifierKeys): Deleted.
(__webpack_require__.DefaultExternalServices.get isInAutomation): Deleted.
(__webpack_require__.DefaultExternalServices.updateEditorStates): Deleted.
(__webpack_require__.const.PDFViewerApplication.async initialize): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _readPreferences): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _parseHashParameters): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializeL10n): Deleted.
(__webpack_require__.const.PDFViewerApplication._forceCssTheme): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializeViewerComponents): Deleted.
(__webpack_require__.const.PDFViewerApplication.run): Deleted.
(__webpack_require__.const.PDFViewerApplication.get initialized): Deleted.
(__webpack_require__.const.PDFViewerApplication.get initializedPromise): Deleted.
(__webpack_require__.const.PDFViewerApplication.zoomIn): Deleted.
(__webpack_require__.const.PDFViewerApplication.zoomOut): Deleted.
(__webpack_require__.const.PDFViewerApplication.zoomReset): Deleted.
(__webpack_require__.const.PDFViewerApplication.get pagesCount): Deleted.
(__webpack_require__.const.PDFViewerApplication.get page): Deleted.
(__webpack_require__.const.PDFViewerApplication.set page): Deleted.
(__webpack_require__.const.PDFViewerApplication.get supportsPrinting): Deleted.
(__webpack_require__.const.PDFViewerApplication.get supportsFullscreen): Deleted.
(__webpack_require__.const.PDFViewerApplication.get supportsIntegratedFind): Deleted.
(__webpack_require__.const.PDFViewerApplication.get supportsDocumentFonts): Deleted.
(__webpack_require__.const.PDFViewerApplication.get loadingBar): Deleted.
(__webpack_require__.const.PDFViewerApplication.get supportedMouseWheelZoomModifierKeys): Deleted.
(__webpack_require__.const.PDFViewerApplication.initPassiveLoading): Deleted.
(__webpack_require__.const.PDFViewerApplication.setTitleUsingUrl): Deleted.
(__webpack_require__.const.PDFViewerApplication.setTitle): Deleted.
(__webpack_require__.const.PDFViewerApplication.get _docFilename): Deleted.
(__webpack_require__.const.PDFViewerApplication._hideViewBookmark): Deleted.
(__webpack_require__.const.PDFViewerApplication._cancelIdleCallbacks): Deleted.
(__webpack_require__.const.PDFViewerApplication.async close): Deleted.
(__webpack_require__.const.PDFViewerApplication.async open): Deleted.
(__webpack_require__.const.PDFViewerApplication._ensureDownloadComplete): Deleted.
(__webpack_require__.const.PDFViewerApplication.async download): Deleted.
(__webpack_require__.const.PDFViewerApplication.async save): Deleted.
(__webpack_require__.const.PDFViewerApplication.downloadOrSave): Deleted.
(__webpack_require__.const.PDFViewerApplication.fallback): Deleted.
(__webpack_require__.const.PDFViewerApplication._documentError): Deleted.
(__webpack_require__.const.PDFViewerApplication._otherError.closeButton.onclick): Deleted.
(__webpack_require__.const.PDFViewerApplication._otherError.moreInfoButton.onclick): Deleted.
(__webpack_require__.const.PDFViewerApplication._otherError.lessInfoButton.onclick): Deleted.
(__webpack_require__.const.PDFViewerApplication._otherError): Deleted.
(__webpack_require__.const.PDFViewerApplication.progress): Deleted.
(__webpack_require__.const.PDFViewerApplication.load.firstPagePromise.then): Deleted.
(__webpack_require__.const.PDFViewerApplication.load.async pagesPromise): Deleted.
(__webpack_require__.const.PDFViewerApplication.load): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _scriptingDocProperties): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _collectTelemetry): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializeAutoPrint): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializeMetadata): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializePageLabels): Deleted.
(__webpack_require__.const.PDFViewerApplication._initializePdfHistory): Deleted.
(__webpack_require__.const.PDFViewerApplication._initializeAnnotationStorageCallbacks): Deleted.
(__webpack_require__.const.PDFViewerApplication.setInitialView): Deleted.
(__webpack_require__.const.PDFViewerApplication._cleanup): Deleted.
(__webpack_require__.const.PDFViewerApplication.forceRendering): Deleted.
(__webpack_require__.const.PDFViewerApplication.beforePrint): Deleted.
(__webpack_require__.const.PDFViewerApplication.afterPrint): Deleted.
(__webpack_require__.const.PDFViewerApplication.rotatePages): Deleted.
(__webpack_require__.const.PDFViewerApplication.requestPresentationMode): Deleted.
(__webpack_require__.const.PDFViewerApplication.triggerPrinting): Deleted.
(__webpack_require__.const.PDFViewerApplication.bindEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication.bindWindowEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication.unbindEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication.unbindWindowEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication.accumulateWheelTicks): Deleted.
(__webpack_require__.const.PDFViewerApplication._unblockDocumentLoadEvent): Deleted.
(__webpack_require__.const.PDFViewerApplication._reportDocumentStatsTelemetry): Deleted.
(__webpack_require__.const.PDFViewerApplication.get scriptingReady): Deleted.
(__webpack_require__.validateFileURL): Deleted.
(__webpack_require__.async loadFakeWorker): Deleted.
(__webpack_require__.async loadPDFBug): Deleted.
(__webpack_require__.reportPageStatsPDFBug): Deleted.
(__webpack_require__.): Deleted.
(__webpack_require__): Deleted.
* Source/WebKit/PdfJSGResources.cmake:

Canonical link: <a href="https://commits.webkit.org/253912@main">https://commits.webkit.org/253912@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/825b0743e5a4f012fba44d1530c525518a065af0

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/87439 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/69/builds/31528 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/18245 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/96667 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/150051 "Built successfully and passed tests") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/64/builds/29893 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/26077 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/79558 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/91438 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/93057 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/68/builds/24152 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/74225 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/23990 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/79120 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/79347 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/67008 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/27603 "Built successfully") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/13179 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/27557 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/14195 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/29245 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/37054 "Passed tests") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/1107 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/29172 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/33473 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Works great, thanks!

mcatanzaro committed:
Update to PDF.js v2.16.105
https://bugs.webkit.org/show_bug.cgi?id=244497

Reviewed by Tim Nguyen.

This also fixes an incremental build problem in PdfJSGResources.cmake.
We need to remove the dependency files when generating the resource
manifests to force them to be refreshed; otherwise, CMake uses the stale
dependency file and complains if any file disappeared when updating
PDF.js.

* Source/ThirdParty/pdfjs/PdfJSFiles.cmake:
* Source/ThirdParty/pdfjs/build/pdf.js:
(return.):
(return):
* Source/ThirdParty/pdfjs/build/pdf.sandbox.js:
* Source/ThirdParty/pdfjs/build/pdf.worker.js:
(return.):
* Source/ThirdParty/pdfjs/web/debugger.css:
(#viewer.textLayer-visible .textLayer span[aria-owns]):
* Source/ThirdParty/pdfjs/web/images/toolbarButton-editorNone.svg: Removed.
* Source/ThirdParty/pdfjs/web/locale/be/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/cs/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/cy/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/de/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/dsb/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/el/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/en-GB/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/en-US/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/eo/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/es-AR/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/es-CL/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/es-MX/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/fi/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/fr/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/fy-NL/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/gn/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/he/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/hsb/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/hu/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/ia/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/id/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/is/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/it/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/ja/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/ka/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/kab/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/kk/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/ko/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/nb-NO/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/nn-NO/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/oc/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/pa-IN/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/pl/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/pt-BR/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/pt-PT/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/ru/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/si/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/sk/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/sl/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/sv-SE/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/tg/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/th/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/tr/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/uk/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/vi/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/zh-CN/viewer.properties:
* Source/ThirdParty/pdfjs/web/locale/zh-TW/viewer.properties:
* Source/ThirdParty/pdfjs/web/viewer.css:
(.annotationLayer .popup):
(.annotationLayer .annotationTextContent):
(.annotationLayer .annotationTextContent span):
(:root):
(#editorNone::before): Deleted.
* Source/ThirdParty/pdfjs/web/viewer.html:
* Source/ThirdParty/pdfjs/web/viewer.js:
(exports.OutputScale):
(exports.OutputScale.prototype.get scaled):
(exports.scrollIntoView):
(exports.const.debounceScroll):
(exports.watchScroll):
(exports.parseQueryString):
(exports.removeNullCharacters):
(exports.binarySearchFirstItem):
(exports.approximateFraction):
(exports.roundToDivide):
(exports.getPageSizeInches):
(exports):
(exports.):
(exports.checkCanvasSizeLimitation): Deleted.
(exports.checkResizeObserver): Deleted.
(exports.AppOptions): Deleted.
(exports.AppOptions.get const): Deleted.
(exports.AppOptions.getAll): Deleted.
(exports.AppOptions.set userOptions): Deleted.
(exports.AppOptions.remove): Deleted.
(exports.AppOptions._hasUserOptions): Deleted.
(__webpack_require__.DefaultExternalServices): Deleted.
(__webpack_require__.DefaultExternalServices.updateFindControlState): Deleted.
(__webpack_require__.DefaultExternalServices.updateFindMatchesCount): Deleted.
(__webpack_require__.DefaultExternalServices.initPassiveLoading): Deleted.
(__webpack_require__.DefaultExternalServices.reportTelemetry): Deleted.
(__webpack_require__.DefaultExternalServices.createDownloadManager): Deleted.
(__webpack_require__.DefaultExternalServices.createPreferences): Deleted.
(__webpack_require__.DefaultExternalServices.createL10n): Deleted.
(__webpack_require__.DefaultExternalServices.createScripting): Deleted.
(__webpack_require__.DefaultExternalServices.get supportsIntegratedFind): Deleted.
(__webpack_require__.DefaultExternalServices.get supportsDocumentFonts): Deleted.
(__webpack_require__.DefaultExternalServices.get supportedMouseWheelZoomModifierKeys): Deleted.
(__webpack_require__.DefaultExternalServices.get isInAutomation): Deleted.
(__webpack_require__.DefaultExternalServices.updateEditorStates): Deleted.
(__webpack_require__.const.PDFViewerApplication.async initialize): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _readPreferences): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _parseHashParameters): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializeL10n): Deleted.
(__webpack_require__.const.PDFViewerApplication._forceCssTheme): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializeViewerComponents): Deleted.
(__webpack_require__.const.PDFViewerApplication.run): Deleted.
(__webpack_require__.const.PDFViewerApplication.get initialized): Deleted.
(__webpack_require__.const.PDFViewerApplication.get initializedPromise): Deleted.
(__webpack_require__.const.PDFViewerApplication.zoomIn): Deleted.
(__webpack_require__.const.PDFViewerApplication.zoomOut): Deleted.
(__webpack_require__.const.PDFViewerApplication.zoomReset): Deleted.
(__webpack_require__.const.PDFViewerApplication.get pagesCount): Deleted.
(__webpack_require__.const.PDFViewerApplication.get page): Deleted.
(__webpack_require__.const.PDFViewerApplication.set page): Deleted.
(__webpack_require__.const.PDFViewerApplication.get supportsPrinting): Deleted.
(__webpack_require__.const.PDFViewerApplication.get supportsFullscreen): Deleted.
(__webpack_require__.const.PDFViewerApplication.get supportsIntegratedFind): Deleted.
(__webpack_require__.const.PDFViewerApplication.get supportsDocumentFonts): Deleted.
(__webpack_require__.const.PDFViewerApplication.get loadingBar): Deleted.
(__webpack_require__.const.PDFViewerApplication.get supportedMouseWheelZoomModifierKeys): Deleted.
(__webpack_require__.const.PDFViewerApplication.initPassiveLoading): Deleted.
(__webpack_require__.const.PDFViewerApplication.setTitleUsingUrl): Deleted.
(__webpack_require__.const.PDFViewerApplication.setTitle): Deleted.
(__webpack_require__.const.PDFViewerApplication.get _docFilename): Deleted.
(__webpack_require__.const.PDFViewerApplication._hideViewBookmark): Deleted.
(__webpack_require__.const.PDFViewerApplication._cancelIdleCallbacks): Deleted.
(__webpack_require__.const.PDFViewerApplication.async close): Deleted.
(__webpack_require__.const.PDFViewerApplication.async open): Deleted.
(__webpack_require__.const.PDFViewerApplication._ensureDownloadComplete): Deleted.
(__webpack_require__.const.PDFViewerApplication.async download): Deleted.
(__webpack_require__.const.PDFViewerApplication.async save): Deleted.
(__webpack_require__.const.PDFViewerApplication.downloadOrSave): Deleted.
(__webpack_require__.const.PDFViewerApplication.fallback): Deleted.
(__webpack_require__.const.PDFViewerApplication._documentError): Deleted.
(__webpack_require__.const.PDFViewerApplication._otherError.closeButton.onclick): Deleted.
(__webpack_require__.const.PDFViewerApplication._otherError.moreInfoButton.onclick): Deleted.
(__webpack_require__.const.PDFViewerApplication._otherError.lessInfoButton.onclick): Deleted.
(__webpack_require__.const.PDFViewerApplication._otherError): Deleted.
(__webpack_require__.const.PDFViewerApplication.progress): Deleted.
(__webpack_require__.const.PDFViewerApplication.load.firstPagePromise.then): Deleted.
(__webpack_require__.const.PDFViewerApplication.load.async pagesPromise): Deleted.
(__webpack_require__.const.PDFViewerApplication.load): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _scriptingDocProperties): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _collectTelemetry): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializeAutoPrint): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializeMetadata): Deleted.
(__webpack_require__.const.PDFViewerApplication.async _initializePageLabels): Deleted.
(__webpack_require__.const.PDFViewerApplication._initializePdfHistory): Deleted.
(__webpack_require__.const.PDFViewerApplication._initializeAnnotationStorageCallbacks): Deleted.
(__webpack_require__.const.PDFViewerApplication.setInitialView): Deleted.
(__webpack_require__.const.PDFViewerApplication._cleanup): Deleted.
(__webpack_require__.const.PDFViewerApplication.forceRendering): Deleted.
(__webpack_require__.const.PDFViewerApplication.beforePrint): Deleted.
(__webpack_require__.const.PDFViewerApplication.afterPrint): Deleted.
(__webpack_require__.const.PDFViewerApplication.rotatePages): Deleted.
(__webpack_require__.const.PDFViewerApplication.requestPresentationMode): Deleted.
(__webpack_require__.const.PDFViewerApplication.triggerPrinting): Deleted.
(__webpack_require__.const.PDFViewerApplication.bindEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication.bindWindowEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication.unbindEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication.unbindWindowEvents): Deleted.
(__webpack_require__.const.PDFViewerApplication.accumulateWheelTicks): Deleted.
(__webpack_require__.const.PDFViewerApplication._unblockDocumentLoadEvent): Deleted.
(__webpack_require__.const.PDFViewerApplication._reportDocumentStatsTelemetry): Deleted.
(__webpack_require__.const.PDFViewerApplication.get scriptingReady): Deleted.
(__webpack_require__.validateFileURL): Deleted.
(__webpack_require__.async loadFakeWorker): Deleted.
(__webpack_require__.async loadPDFBug): Deleted.
(__webpack_require__.reportPageStatsPDFBug): Deleted.
(__webpack_require__.): Deleted.
(__webpack_require__): Deleted.
* Source/WebKit/PdfJSGResources.cmake:

Canonical link: https://commits.webkit.org/253912@main

zdobersek commented on Source/WebKit/PdfJSGResources.cmake:
```diff
@@ -5,6 +5,7 @@ macro(WEBKIT_BUILD_PDFJS_GRESOURCES _derived_sources_dir)
     add_custom_command(
         OUTPUT ${_derived_sources_dir}/PdfJSGResourceBundle.xml
         DEPENDS ${TOOLS_DIR}/glib/generate-pdfjs-resource-manifest.py ${PdfJSFiles}
+        COMMAND ${CMAKE_COMMAND} -E rm -f ${_derived_sources_dir}/PdfJSGResourceBundle.deps
```
This `rm` command support was added in CMake 3.17. Minimum required version ATM is 3.12, so this ends up failing on older versions of CMake.

mcatanzaro replied:
Drat

mcatanzaro replied:
I thought I was being clever and cross-platform friendly by doing this instead of normal `rm`. Alas. I'll change it to fall back to normal `rm` from PATH for older versions of CMake. That will fail on Windows, but (a) this file is currently only used by GTK and WPE, which don't work on Windows, and (b) people compiling for Windows can upgrade to newer CMake versions to avoid this.

mcatanzaro replied:
Turns out `cmake -E rm` is a new replacement for `cmake -E remove`. The `-E remove` is deprecated, but the behavior difference does not matter in this case, so I'll switch to that.

mcatanzaro replied:
https://bugs.webkit.org/show_bug.cgi?id=244728
